roles/edge-image-builder/tasks/main.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Create eib images directory
+  become: true
+  file:
+    path: "{{ eib_path }}/images"
+    state: directory
+
+- name: Copy OS image to eib images directory
+  become: true
+  copy:
+    src: "{{ os_image_path }}/{{ os_image_name }}"
```

Comment:
We should default these in the role defaults, and probably add a validation task to exit if they are not defined and/or the file doesn't exist?

Reply:
This is part of the image-cache block, in order to avoid importing the role if the image is not present. Failure in the first point approach